spec/lib/active_versioning/model/versioned_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'spec_helper'
+
+RSpec.describe ActiveVersioning::Model::Versioned do
+  let!(:user) { ActiveVersioning::Test::User.create(name: 'Bo Janglez', email: 'bo.janglez@gmail.com') }
+
+  subject { ActiveVersioning::Test::Post.create(title: 'So Post', body: 'Such interesting.  Very wow.', user: user) }
+
+  it { should have_many(:versions) }
+
+  describe "after create" do
+    it "creates a new version" do
+      expect {
+        ActiveVersioning::Test::Post.create(title: 'So Post', body: 'Such interesting.  Very wow.', user: user, version_author: user)
+      }.to change { ActiveVersioning::Test::Version.count }.by 1
+    end
+  end
+
+  describe "#version_author" do
+    it "should be able to set and view the author" do
+      subject.version_author = user
+
+      expect(subject.version_author).to eq user
+    end
+  end
```

Comment:
When a versioned object is saved, the `version_author` is pulled off the record and stored with the version as the `committer`.  Yeah, this test is super simple and just makes sure the necessary methods are still there.